lib/content-services/src/lib/tag/tag-node-list.component.scss

```diff
@@ -1,7 +1,31 @@
 .adf-tag-node-list {
+    display: flex;
+    align-items: center;
+    flex-direction: row;
+    width: inherit;
+
+    &.adf-flex-column {
+        flex-direction: column !important;
```

Comment:
Is that !important required? Can't we do it without it?

Reply:
I just checked and we can remove it, thanks!